src/Application/Common/Constants.cs

```diff
@@ -24,6 +24,7 @@ public class Constants
     public float ItemBreakChance { get; set; }
     public int BrokenItemRepairPenaltySeconds { get; set; }
     public float[] ItemSellCostCoefs { get; set; } = Array.Empty<float>();
+    public float[] ItemReforgeCostCoefs { get; set; } = Array.Empty<float>();

```

Comment:
The CrpgConstants in Module.Server should be updated too

Reply:
It's already the case. Did you miss it , or am i missing something?